Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
Procfile is required for foreman and Heroku; does this collide with config/puma.rb ?